base/dict.jl

```diff
@@ -671,6 +672,7 @@ function _delete!(h::Dict{K,V}, index) where {K,V}
     h.ndel += ndel
     h.count -= 1
     h.age += 1
+    h.maxprobe = max(h.maxprobe, h.size-1)
```

Comment:
Is size a field of Dict? Do you mean count?

I don't think that's right either since I think the max probe can be larger than the count.